shell/common/shell_io_manager.h

```diff
@@ -69,7 +69,7 @@ class ShellIOManager final : public IOManager {
   // Unref queue management.
   fml::RefPtr<flutter::SkiaUnrefQueue> unref_queue_;
 
-  const std::shared_ptr<const fml::SyncSwitch>& is_gpu_disabled_sync_switch_;
```

Comment:
This was your problem, you shouldn't be storing a const reference, you should just be passing by const reference in the argument. =)

Reply:
ahhhh ok. I think it's probably fine to just let callers who want to avoid the copy do a `std::move` though...  If you want me to do a follow up I can.